examples/tf-azurerm-computegroup/main.tf

```diff
@@ -0,0 +1,122 @@
+resource "azurerm_resource_group" "rg" {
+  name     = "${var.resource_group_name}"
+  location = "${var.location}"
+  tags     = "${var.tags}"
+}
+
+resource "azurerm_virtual_network" "vnet" {
+  name                = "acctvn"
+  address_space       = ["10.0.0.0/16"]
+  location            = "${var.location}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+}
+
+resource "azurerm_subnet" "subnet1" {
+  name                 = "acctsub"
+  resource_group_name  = "${azurerm_resource_group.rg.name}"
+  virtual_network_name = "${azurerm_virtual_network.vnet.name}"
+  address_prefix       = "10.0.1.0/24"
+}
+
+resource "azurerm_public_ip" "publicip" {
+  name                         = "test"
```

Comment:
could I suggest this becomes `${var.resource_group_name}-pip` to make this more obvious?